release.sh

```diff
@@ -112,6 +100,7 @@ git add README.md
 git commit -m "Version bump from ${OLD_VERSION} to ${NEW_VERSION}"
 git tag -s "v${NEW_VERSION}" -m "Making release ${NEW_VERSION}"
 # only ensure master is updated if it is a non-alpha release
+exit
```

Comment:
I don't think we want this `exit` here.

Reply:
Oh no!  It was to prevent any accidents :)